### ISSUE TITLE
Model Architecture

### ISSUE THREAD

**Issue by g3mini28** [omoindrot/tensorflow-triplet-loss#39]
Hi,

I am currently new to all this and still learning. Is your implementation very similar to FaceNet approach? As I am currently working on a facial recognition project and I am trying FaceNet approach. I have read David Sandberg's on FaceNet and noticed he used a model architecture of inception ResNet v1 to train the dataset. 

May I know what is the model architecture that you are using and if I were to use your approach, must I retrain a huge dataset of photos and use this pretrained model, to train my own much smaller dataset? Any suggestions of what I need to do? Thank you.

**Comment by omoindrot** [comment]
Since my example works with MNIST, I'm not using a very big architecture (just 2 convolutions + fully connected).

For face recognition, you need a bigger model like resnet. If your dataset is small, you may want to start with a pretrained model and fine-tune it, or even freeze most of the layers.
